### PR TITLE
Move to support 64-bit ruby on Windows.

### PR DESCRIPTION
**Pull request by ksubrama**
This will allow omnibus itself to run on 64-bit ruby.

### CODE REVIEW

**Comment by jaym** [comment]
why do we need omnibus running in a 64 bit ruby?

**Comment by ksubrama** [comment]
I guess it's not a priority...  but I'm assuming we're going to want to build on Nano eventually...

File: omnibus.gemspec [hunk 1]
```diff
@@ -23,9 +23,9 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'chef-sugar',       '~> 3.0'
   gem.add_dependency 'cleanroom',        '~> 1.0'
-  gem.add_dependency 'mixlib-shellout',  '~> 1.4'
+  gem.add_dependency 'mixlib-shellout',  '~> 2.0'
   gem.add_dependency 'mixlib-versioning'
-  gem.add_dependency 'ohai',             '~> 7.2'
+  gem.add_dependency 'ohai',             '~> 8.2'
```
Comment (schisamo) [inline omnibus.gemspec]: Let's make this:

```ruby
gem.add_dependency 'ohai',             '~> 8.0'
```

Comment (ksubrama) [inline omnibus.gemspec]: Done!

**Comment by danielsdeleo** [comment]
:+1:

**Comment by lamont-granquist** [comment]
:+1:

**Pull request merged**